lua/entities/ent_jack_gmod_ezatmine.lua

```diff
@@ -9,6 +9,7 @@ ENT.NoSitAllowed=true
 ENT.Spawnable=true
 ENT.AdminSpawnable=true
 ---
+ENT.EZScannerDanger=true
```

Comment:
can we do the acronym-away camelCasing that we do everywhere else in the codebase?

EZscannerDanger

Reply:
Sure.
Well now I know how it's done.
CTRL+SHIFT+F time.

Reply:
Replaced now.